src/modules/ipfix/aggregator/AggregatorBaseCfg.cpp

```diff
@@ -46,8 +46,6 @@ AggregatorBaseCfg::AggregatorBaseCfg(XMLElement* elem)
 			// get the time values or set them to '0' if they are not specified
 			maxBufferTime = getTimeInUnit("activeTimeout", SEC, 0, e);
```

Comment:
Should we rename this variable to `activeTimeout` to be consistent with IPFIX terminology?